include/seqan3/alphabet/concept.hpp

```diff
@@ -618,7 +618,7 @@ void alphabet_size(args_t ...) = delete;
 template <typename alph_t>
 struct alphabet_size_fn
 {
-private:
+public:
```

Comment:
These are weird, but oh well.

Reply:
But do we know why this is necessary? I mean I don't see right now why the private CPO members are not accessible from within the class itself?

Reply:
clang allows this, gcc not. Not sure what the standard says, I would expect it to work, but it doesn't.

Reply:
I asked on the concepts [slack](https://cpplang.slack.com/archives/C6K47F8TT/p15674338610107000), but none answered.

Reply:
It works with GCC10 with c++20 concepts and concepts-ts. I would consider it a bug of GCC<=9, but one that likely won't be fixed.
Nothing to be done about it.

Reply:
(You can WORKAROUND declare it for GCC<=9 but that arguably makes the code worse that just making it public, I think)

Reply:
okay, good to know

Reply:
💥